_posts/2018-11-22-planning-session.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "Planning Session"
+slug: planning-session
+text: 'Winter planning session for Toronto Mesh'
+location: HackLab.TO, 1266 Queen St
+locationLink: https://www.openstreetmap.org/node/3522641496
+date: 2018-11-22
+startTime: '18:30'
+endTime: '21:30'
+---
+
+This us Toronto Mesh's seventh planning session. We will regroup and plan for the winter and beyond! During this session we will:
+- Review all projects and efforts undertaken since April.
+- Re-evaluate our long-term goals.
+- Set the priorities and projects to work on for the next six months.
+- Assign working group leads for the next six months. You can see our current [working groups and governance processes here](https://github.com/tomeshnet/documents/blob/master/governance/coordination-structure.md)!
+
+We will meet at {{page.startTime | date:"%-I:%M %P "}} in [{{ page.location }}]({{ page.locationLink }}).
+
+This event has a [code of conduct](/code-of-conduct/).
+
+New to Toronto Mesh?  
```

Comment:
Maybe this could be a heading or bolded?

Reply:
Agreed! I'll update it.